src/object/authenticate.h

```diff
@@ -138,6 +138,21 @@ MOP au_get_dba_user (void);
     } \
   while (0)
 
+/* 
+ * name is user_specified_name.
+ * owner_name must be a char array of size DB_MAX_IDENTIFIER_LENGTH to copy user_specified_name.
+ * class_name refers to class_name after dot(.).
+ */
+#define SPLIT_USER_SPECIFIED_NAME(name, owner_name, class_name) \
+	do \
+	  { \
+	    memset ((owner_name), '\0', DB_MAX_IDENTIFIER_LENGTH); \
```

Comment:
This is unnecessary code.

Reply:
Sometimes it is called in a loop. In this case, memset () is required.

Reply:
It is checked with unnecessary memset() and will be removed.